_includes/master/charts/calico/templates/calico-kube-controllers.yaml

```diff
@@ -84,49 +85,7 @@ spec:
           secret:
             secretName: calico-etcd-secrets
             defaultMode: 0400
-
----
-
-apiVersion: v1
-kind: ServiceAccount
```

Comment:
Did this service account get lost?

Reply:
No I re-arranged the template a bit so that the common parts were not repeated in an effort to ensure an update for one version will be reflected in the other.  So now this Service account is included for both flavors with the one at the bottom of the file.